src/components/fields/location-field/location-helper.ts

```diff
@@ -143,6 +143,28 @@ export namespace LocationHelper {
 		}
 	};
 
+	export const checkAndSetPinLocationAsResult = (query: string): IResultsMetaData => {
+		const [lat, lng] = query
+			.split(":")[1]
+			.split(",")
+			.map((value) => parseFloat(value.trim()));
+		let parsedResult = [];
+		if (LocationHelper.isCoordinateInBounds({ lat: lat, lng: lng })) {
+			parsedResult = [
+				{
+					address: query,
+					lat: lat,
+					lng: lng,
+				},
+			];
+		}
```

Comment:
just suggesting some shorthands to use
```suggestion
		if (LocationHelper.isCoordinateInBounds({ lat, lng })) {
			parsedResult = [
				{
					address: query,
					lat,
					lng,
				},
			];
		}
```